### ISSUE TITLE
Repository size is zero

### ISSUE THREAD

**Issue by marcheimendinger** [ForkIssues/Tracker#868]
Strange issue but there is maybe a technical reason about it : all my private repositories of GitHub have a displayed size of zero (even if they are not empty). On a local instance of GitLab (not gitlab.com) only one repo has this problem, even if I have multiple private ones.

**Comment by DanPristupov** [comment]
Do you mean the size in the repository manager?

What is the result of `git count-objects --verbose` command for such a repository?
This command returns output in the following format:

```
count: 5254
size: 31764
in-pack: 39983
packs: 20
size-pack: 36543
prune-packable: 48
garbage: 0
size-garbage: 0
```

The `size-pack` should be the repository size (in KB). This is the value displayed in the repository details in repo manager.

**Comment by marcheimendinger** [comment]
Result of `git count-objects --verbose` :
```
count: 925
size: 36804
in-pack: 0
packs: 0
size-pack: 0
prune-packable: 0
garbage: 0
size-garbage: 0
```

And that's what Fork displays :
![Screenshot 2020-01-09 at 09 08 16](https://user-images.githubusercontent.com/43568598/72049627-951bf080-32bf-11ea-88bc-ee2c222f0b7d.png)

**Comment by DanPristupov** [comment]
Probably we should use the 'size' value in a case 'size-pack' is 0.

**Comment by marcheimendinger** [comment]
`git repack` did the trick apparently.

**Comment by DanPristupov** [comment]
What is the result of the `git count-objects --verbose` now?

**Comment by marcheimendinger** [comment]
```
count: 925
size: 36804
in-pack: 925
packs: 1
size-pack: 34244
prune-packable: 925
garbage: 0
size-garbage: 0
```

**Comment by DanPristupov** [comment]
I see. We should probably use 'size' in a case 'size-pack' is 0.

**Comment by DanPristupov** [comment]
Fixed in 1.0.89
